lib/scenic/adapters/postgres/index_reapplication.rb

```diff
@@ -35,6 +35,18 @@ def on(name)
           indexes.each(&method(:try_index_create))
         end
 
+        def on_side_by_side(name, new_table_name, temporary_id)
+          indexes = Indexes.new(connection: connection).on(name)
+          indexes.each_with_index do |index, i|
+            old_name = "predrop_index_#{temporary_id}_#{i}"
```

Comment:
This looks like it cuts 24 characters off the alloted 63 for postgres identifiers (see earlier comment). We may need to think more about how to lessen the chance this is an issue for folks

Reply:
This one doesn't actually have the index name in it (it just looks like `predrop_index_1705535993_1`). I'm pretty sure I did it that way _because_ I hit the index size limit. :-D